DOC-600 Add quest_db output released in v4.37.0 (#70)
redpanda-data/rp-connect-docs: modules/components/pages/outputs/questdb.adoc

= questdb
:type: output
:page-beta: true
:categories: ["Services"]

// tag::single-source[]

// © 2024 Redpanda Data Inc.

component_type_dropdown::[]

> Pushes messages to a QuestDB table.

Reviewer: ```suggestion
Pushes messages to a https://questdb.io/docs/[QuestDB^] table.
```

Author: Updated as requested.

    doubles: [] # No default (optional)
    error_on_empty_messages: false
```
--
Advanced::
+
--
```yml
# All configuration fields, showing default values
output:
  label: ""
  questdb:
    max_in_flight: 64
    batching:
      count: 0
      byte_size: 0
      period: ""
      check: ""
      processors: [] # No default (optional)
    tls:
      enabled: false
      skip_cert_verify: false
      enable_renegotiation: false
      root_cas: ""
      root_cas_file: ""
      client_certs: []
    address: localhost:9000 # No default (required)
    username: "" # No default (optional)
    password: "" # No default (optional)
    token: "" # No default (optional)
    retry_timeout: "" # No default (optional)
    request_timeout: "" # No default (optional)
    request_min_throughput: 0 # No default (optional)
    table: trades # No default (required)
    designated_timestamp_field: "" # No default (optional)
    designated_timestamp_unit: auto
    timestamp_string_fields: [] # No default (optional)
    timestamp_string_format: Jan _2 15:04:05.000000Z0700
    symbols: [] # No default (optional)
    doubles: [] # No default (optional)
    error_on_empty_messages: false
```
--
======

> IMPORTANT: Redpanda recommends that you enable the dedupe feature on the QuestDB server. For more information about about deploying, configuring, and using QuestDB, see the https://questdb.io/docs/[QuestDB documentation]. 

Reviewer: ```suggestion
IMPORTANT: Redpanda Data recommends enabling the dedupe feature on the QuestDB server. For more information about deploying, configuring, and using QuestDB, see the https://questdb.io/docs/[QuestDB documentation^]. 
```

Author: Updated as requested.
